contracts/Market.sol

```diff
@@ -470,6 +468,10 @@ contract FIFOMarket is
       this.totalReservedSupply(); // todo store reserved amount in removal data instead
```

Comment:
I think there's a problem with this because `totalReservedSupply()` is giving you the total number of NRTs that are reserved, not the number of individual distinct removal token ids, which is what `numberOfTokensOwnedByAddress` is giving you, so these apples and oranges shouldn't be being subtracted from each other

Reply:
fixed this in our pair branch from today